### DISCUSSION TITLE
Update and SSL Certificate

### DISCUSSION THREAD

**Discussion by rinjuu123** [dyc3/opentogethertube#1446] [Q&A] [answered]
Hello,

how do i update OTT?
and how do i setup a SSL certificate?

**Comment by dyc3** [comment, marked as answer]
If you're using the docker image (which you should), all you need to do is run:
```bash
docker pull dyc3/opentogethertube:latest
```
and the next time you start the container, it'll use the latest version.

As for setting up SSL, that somewhat depends on whatever you're using as a reverse proxy (probably nginx or apache). If you have a domain name, you can get an SSL cert from Lets Encrypt using `certbot-auto`. See this for an example of an nginx config: https://github.com/dyc3/opentogethertube/blob/master/docs/how-to-deploy.md#reverse-proxy

**Reply by rinjuu123** [reply]
Thank you, that helped alot!
